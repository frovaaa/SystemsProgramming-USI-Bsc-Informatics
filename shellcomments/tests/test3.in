#!/bin/sh

##
## This is a comment
##

This is not
