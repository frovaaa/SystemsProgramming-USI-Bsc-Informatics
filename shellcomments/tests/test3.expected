!/bin/sh
#
# This is a comment
#
